docker/terra-node-ci/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM cerner/terra-node-base:latest
+
+# Copy all the things
+ONBUILD COPY . /opt/module
+
+ONBUILD RUN npm install --loglevel error
```

Comment:
Do we need this Dockerfile? Why not use the base image directly?

Reply:
Did we drop the command: https://github.com/cerner/terra-toolkit/blob/master/docker/terra-node/Dockerfile#L26

Reply:
Yes we need this docker container, instead of combining dev and ci into one container (dev doesn't need npm install or to copy the files, ci doesn't need zsh) i split them out into child docker containers.

yes I removed the npm install command, it's not needed with the separate docker containers.